sonar-analysis/action.yml

```diff
@@ -101,16 +110,28 @@ runs:
         elif [[ ${{ steps.extract_branch.outputs.branch }} == ${{ inputs.primary_release_branch }} ]]; then
           echo "Executing an analysis on the main branch"
           mvn -B -s ${{ inputs.mvn_settings_filepath }} org.owasp:dependency-check-maven:7.1.1:aggregate sonar:sonar \
```

Comment:
shouldn't org.owasp:dependency-check-maven:7.1.1:aggregate be removed here - or do we still want to keep a job where both checks are done at the same time?

Where are we then calling the two different checks one after the other?
org.owasp:dependency-check-maven:7.1.1:aggregate checks the library dependencies and creates a report in local files
sonar:sonar does the sonar static code analysis, but also picks up the locally cached report files from the previously run dependency check to include them in the sonar report

Reply:
This was indeed something I missed when doing the PR.